src/verifiers/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 add_subdirectory(nullverifier)
 add_subdirectory(sgx-ecdsa)
-add_subdirectory(sev-snp)
+
+if(HOST)
```

Comment:
if((EXISTS "/usr/include/linux/sev-guest.h") AND HOST)

Reply:
The compilation process of sev-snp does NOT rely on `sev-guest.h`

Reply:
ok, got it